### ISSUE TITLE
iOS Switching Orientation

### ISSUE THREAD

**Issue by BadScooter1980** [zurb/foundation-sites#11182]
There seems to be an issue with Foundation 6.4.2 out of the bag.

If you zoom in/out at any time, and then switch to landscape/portrait mode the screen doesn't reset itself again.

I've tested this across multiple browsers and am getting the same result

**Comment by DanielRuf** [comment]
Please provide a codepen which demonstrates this.

**Comment by BadScooter1980** [comment]
https://codepen.io/BadScooter1980/pen/xWvYyo

If you view that on mobile, pinch the screen to zoom in/out, then switch orientation you'll see it doesn't reset itself.

**Comment by DanielRuf** [comment]
Do normal websites reset in this case?

**Comment by BadScooter1980** [comment]
Yes, they do. It seems like Foundation isn't registering when the screen has been zoomed back to its normal size.

I just tried on my own website and if I zoom in then out again, change orientation, it works just fine.

**Comment by DanielRuf** [comment]
Which iOS / mobile Safari version is affected?
